source/Octopus.Tentacle.Contracts/Legacy/ReMappedLegacyTypes.cs

```diff
@@ -34,7 +35,8 @@ internal ReMappedLegacyTypes(params string[] nameSpaces)
                 }
             }
 
-            FullNameOfTypesToRemap = set;
```

Comment:
Hashset might be thread safe when used in read only, but immutable hashset is safer.